code/modules/mob/living/carbon/carbon.dm

```diff
@@ -429,6 +429,9 @@
 	return ..()
 
 /mob/living/carbon/proc/vomit(lost_nutrition = 10, blood = FALSE, stun = TRUE, distance = 1, message = TRUE, toxic = FALSE)
+	if ( istype( dna.species, /datum/species/ipc ) ) // NSV13 IPCs do not have stomachs 
```

Comment:
You probably want to make this more general or make a trait out of it (NO_VOMIT). Bit of a patchwork solution to attach it to isipc, essentially.